pkg/skaffold/deploy/kpt/kpt_test.go

```diff
@@ -120,7 +120,7 @@ func TestKpt_Deploy(t *testing.T) {
 				AndRunOut(fmt.Sprintf("kpt fn sink %v", tmpKustomizeDir), ``).
 				AndRunOut(fmt.Sprintf("kustomize build %v", tmpKustomizeDir), ``).
 				AndRunOut("kpt fn run --dry-run", testPod).
-				AndRun("kpt live apply valid_path"),
+				AndRun("kpt live apply valid_path --context kubecontext --namespace testNamespace"),
```

Comment:
Can we add a testcase for `k.kubeConfig` not null?

Reply:
Done. Added a new test